src/networking.c

```diff
@@ -828,6 +828,11 @@ void setDeferredPushLen(client *c, void *node, long length) {
     setDeferredAggregateLen(c,node,length,'>');
 }
 
+inline void utoa(char *buf, uint32_t val, int len) {
+	for(; val && len ; --len, val /= 10)
+		buf[len - 1] = "0123456789"[val % 10];
+}
```

Comment:
missing some comment, specifically since it doesn't have a return value or puts a null terminator, so it's not a standard itoa in that respect.
i suppose you want to avoid the null term, and you only really expect up to 4 characters, which is why you don't use ull2string?

i guess this is all acceptable, maybe with the addition of:
1. rename the function to set it apart from itoa
2. add a comment describing it. (p.s. use block comments, not line comments)
3. use 4 spaces, no tabs